apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx

```diff
@@ -77,6 +77,7 @@ interface ApplicationWrapperCertificatesPropsInterface extends TestableComponent
     hidden: boolean;
     readOnly: boolean;
     triggerSubmit?: boolean;
+    canDiscardCerticate?: () => boolean;
```

Comment:
Correct the spellings of this variable

```suggestion
    canDiscardCertificate?: () => boolean;
```